01_install_requirements.sh

```diff
@@ -43,6 +42,9 @@ sudo yum -y install \
   wget \
   jq
 
+# moved to after all called software is installed
+source common.sh
```

Comment:
The problem with this is we set up logging in common.sh, so if anything fails before this, it's not logged (which will be bad, particularly in CI)

Perhaps we can break that into a separate file, e.g common_logging.sh and source that here, then either include that via common.sh, or update all the scripts that need it to source both files?

cc @stbenjam

Reply:
> The problem with this is we set up logging in common.sh, so if anything fails before this, it's not logged (which will be bad, particularly in CI)

It shouldn't be a problem for CI as we also log the console which will get any errors in this script, it would be more of a problem for people trying to figure out what happened in their own env when scrollback is gone.

> 
> Perhaps we can break that into a separate file, e.g common_logging.sh and source that here, then either include that via common.sh, or update all the scripts that need it to source both files?
> 
> cc @stbenjam